### DISCUSSION TITLE
How to change the parallelism limit to more than 10？

### DISCUSSION THREAD

**Discussion by yuncengshangdepingyuan** [langgenius/dify#9894] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

When I have more than 10 or more parallels, it stops running and can't pass arguments backwards. How can I change this limit to 20 or more?
<img width="776" alt="屏幕截图 2024-10-26 154526" src="https://github.com/user-attachments/assets/a6cb0e1b-3da2-4efe-b11d-dc821f5ac9e1">

**Comment by yuncengshangdepingyuan** [comment]
I find it in the code.but I can't find it in dify\docker\.env.example.can I change it?What are the bad effects?
<img width="634" alt="1" src="https://github.com/user-attachments/assets/a384e9ba-9823-4f24-8982-dc4f9dfa7e6d">

**Comment by coddderX** [comment]
mark

**Comment by pr-maia** [comment]
Relevant to me.

**Comment by hehemagic** [comment]
mark
